R/promptAndRun.R

```diff
@@ -6,9 +6,13 @@ promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
   if (mydir == ".") {
     loopRuns(".", user = user)
   } else if (mydir == "") {
-    dirs <- list.dirs(".", recursive = FALSE)
-    chosendirs <- gms::chooseFromList(dirs, type = "folders")
-    loopRuns(if (length(chosendirs) == 0) "exit" else chosendirs, user = user)
+    if (all(file.exists(c("full.gms", "log.txt", "config.Rdata", "prepare_and_run.R")))) {
+      loopRuns(".")
```

Comment:
should this contain the `user = user` bit?

Reply:
I think it doesn't matter, as calling `rs2` without a directory implies that user is `NULL` and https://github.com/pik-piam/modelstats/blob/master/R/loopRuns.R#L19 will catch it, but I added it anyway.